tests/Unit/NumericalAlgorithms/Interpolation/Test_CubicSpline.cpp

```diff
@@ -100,7 +104,7 @@ void test_with_polynomial(const size_t number_of_points,
 SPECTRE_TEST_CASE("Unit.Numerical.Interpolation.CubicSpline",
                   "[Unit][NumericalAlgorithms]") {
   test_with_polynomial(10, 1, 1.e-14, 1.e-14);
-  test_with_polynomial(10, 2, 1.e-1, 1.e-3);
+  test_with_polynomial(10, 2, 1.e-1, 1.e-2);
```

Comment:
I am still worried about why this tolerance is so high.  Can you plot the error as a function of N between 10 and 100?

Otherwise I am happy for you to squash if @nilsdeppe and @wthrowe agree

Reply:
![Unknown-21](https://user-images.githubusercontent.com/746230/64899068-e9082700-d689-11e9-91c1-0698ddf5a385.png)

Reply:
you're probably using the wrong derivatives at the end points. I.e. the spline assumes zero derivs since it's natural but the underlying function doesn't have that

Reply:
Can you please check that the spline works correctly if the derivatives at the end points are zero.

Reply:
good idea 👍

Reply:
pushed a fixup, please take a look